ehrql/tables/raw/tpp.py

```diff
@@ -659,14 +659,6 @@ class covid_therapeutics_raw(EventFrame):
             date or a past date at the time of form submission.
         """,
     )
-    CASIM05_date_of_symptom_onset = Series(
```

Comment:
There's also the place where these are defined in the QueryTable SQL (backends/tpp.py); if we're not including them in the queryable tables, we don't need to extract them from the db either.

Reply:
oh, yeah, true, removed.